thewalrus/fock_gradients.py

```diff
@@ -15,13 +15,16 @@
 Fock gradients of Gaussian gates
 ================================
 
+**Module name:** :mod:`thewalrus.fock_gradients`
```

Comment:
out-of-scope thought: we should rename this module to simply `thewalrus.fock` (although this would be a breaking change for SF).

The reason being, I was helping a user use TW, and both the user and myself completely overlooked this module, as we assumed it _only_ provided gradients! Whereas the focus of this module is more Fock representations (of which gradients are included)

Reply:
Sounds reasonable, I'd say. 👍